Dfc.CourseDirectory.Services.Tests/BulkUploadService/Apprenticeship/ApprenticeshipCsvStructure.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+
+namespace Dfc.CourseDirectory.Services.Tests.BulkUploadService.Apprenticeship
+{
+    internal static class ApprenticeshipCsvStructure
+    {
+        public static readonly List<string> Fields = new List<string>
```

Comment:
```suggestion
        public static readonly IReadOnlyList<string> Fields = new List<string>
```

Reply:
Ah, fancy, I haven't looked at the new types much. Thanks! Done